client/reader/site-stream/index.jsx

```diff
@@ -127,10 +128,6 @@ const SiteStream = React.createClass( {
 			title = this.translate( 'Loading Site' );
```

Comment:
I recall that during my trial I was working on more discoverable **feed** URLs (https://wordpress.com/read/feeds/developer.wordpress.com), but not on more discoverable **blog** URLs. 

I have summarized the work on this in the latest comment of the related post in the Reader Squad p2 (`p5PDj3-rS-p2`), but I'm willing to shed some light, if I can.